airbyte-config/models/src/main/resources/types/ReplicationAttemptSummary.yaml

```diff
@@ -9,17 +9,36 @@ required:
   - bytesSynced
   - startTime
   - endTime
-additionalProperties: false
+  - totalRecordsEmitted
+  - streamNameToRecordsEmitted
+  - streamNameToBytesEmitted
+  - totalStateMessagesEmitted
+additionalProperties: true
 properties:
   status:
     "$ref": ReplicationStatus.yaml
-  recordsSynced:
+  recordsSynced: # TODO (parker) remove in favor of totalRecordsEmitted
     type: integer
     minValue: 0
-  bytesSynced:
+  bytesSynced: # TODO (parker) should we also remove this in favor of a better-named 'totalBytesEmitted' field?
     type: integer
     minValue: 0
   startTime:
     type: integer
   endTime:
     type: integer
+  totalRecordsEmitted:
+    type: integer
+    minValue: 0
+  streamNameToRecordsEmitted:
+    type: object
```

Comment:
I think you can define this as a map as opposed to just an object in jsonschema so that you can still get type safety in java. maybe something like this? https://stackoverflow.com/a/40750892. Also fine if we just store it as a list of objects (e.g. `[{ streamName: "users", recordsEmitted: 10}, { streamName: "accounts", recordsEmitted: 200}, ...]`